src/pages/index.tsx

```diff
@@ -35,63 +37,68 @@ const Home: NextPage = () => {
   const upgradeVersionOptions = useMemo(() => {
     if (!currentVersion) return versionOptions;
     const [major, minor, patch] = currentVersion.split(".");
-    const filteredVersions = Object.keys(versionOptions)
-      .filter((majorVersion) => Number(majorVersion) >= Number(major))
-      .reduce((acc, majorVersion) => {
-        if (Number(majorVersion) === Number(major)) {
-          const initialValues = versionOptions[majorVersion];
-          const filterValue = (version: string) => {
-            const [, versionMinor, versionPatch] = version.split(".");
-            if (Number(versionMinor) > Number(minor)) return true;
-            if (
-              Number(versionMinor) === Number(minor) &&
-              Number(versionPatch) > Number(patch)
-            )
-              return true;
-            return false;
-          };
-          acc[majorVersion] = initialValues?.filter(filterValue) || [];
+
+    // filter out versions that are older than the current version
+    const filteredVersions = versionOptions
+      .filter((version) => Number(version.major) >= Number(major))
```

Comment:
maybe rename this to option instead of version so we can write option.major and option.versions (instead of version.versions)

Reply:
Done